Dockerfile

```
@@ -1,20 +1,30 @@
 # Copyright 2022-present Open Networking Foundation
+# Copyright 2023-present Intel Corporation
 #
 # SPDX-License-Identifier: Apache-2.0
 #
 
-FROM golang:1.21.3-bookworm AS lb
+FROM golang:1.21.3-bookworm AS builder
 
 LABEL maintainer="ONF <omec-dev@opennetworking.org>"
 
-RUN apt-get update
-RUN apt-get -y install vim 
 RUN cd $GOPATH/src && mkdir -p sctplb
 COPY . $GOPATH/src/sctplb
-RUN cd $GOPATH/src/sctplb && go mod tidy && go install 
+RUN cd $GOPATH/src/sctplb && go mod tidy && CGO_ENABLED=0 go install
```

@thakurajayL, what do you think about this change? So, the source code is aligned to what is executed/used inside the Dockerfile. That is, if a developer adds or updates a dependency, the developer needs to run `go mod tidy` to update `go.mod` and `go.sum` files and these updates files are used in Dockerfile
```suggestion
RUN cd $GOPATH/src/sctplb && CGO_ENABLED=0 go install
```

---

I think proposal looks ok and we can depend on author to update dependency and we should avoid doing it during the build time.

---

It is not about depending on the author but mainly to not hide the dependency update from a PR